apps/passport-client/src/worker/service-worker.ts

```diff
@@ -243,6 +259,16 @@ self.addEventListener("fetch", (event: FetchEvent) => {
         return stableResp;
       }
 
+      // Check Z cache next. We fetch if and only if we do not get a hit.
+      const zResp = await checkZCache(event.request);
+      if (zResp) {
+        swLog.V(`cache hit fetching ${event.request?.url}`);
+        return zResp;
+      } else {
```

Comment:
I missed this on my initial review, but @ax0 can you explain what this else clause is for?  VSCode is showing me that the remainder of this function is all unreachable.  This feels like it might've been debug code meant to bypass functionality.